.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: node_js
 before_install:
-  - npm -g install npm@latest-4
+  - npm -g install npm@6
 node_js:
+  - "10"
```

Comment:
Since node v6 has been deprecated, maybe we have 8, 10, 12 instead?

Reply:
Will do :+1: